rfc/0054-sdk3-management-apis.md

```diff
@@ -3599,11 +3653,18 @@ interface CollectionSpec {
     String ScopeName();
 
     Duration MaxExpiry();
+    
+    Bool History();
 }
 ```
 
 `MaxExpiry` is the time in seconds for the TTL for new documents in the collection. Omit if not set.
 
+`History` is whether history retention override is enabled on this collection. Omit if not set - not set will default to bucket level setting.
+* On Create/Update this setting should be guarded by the bucket capability of `nonDedupedHistory`
+* If the bucket capability is not supported then a `FeatureNotAvailableException` must be raised.
+  * Suggested supplementary text: "history is unsupported by the server, are you using a bucket with the magma storage engine?"
```

Comment:
The suggested text makes it a little ambiguous whether the feature is, or is explicitly not, supported on Magma.